src/qt/sendcoinsentry.cpp

```diff
@@ -250,13 +250,13 @@ bool SendCoinsEntry::updateLabel(const QString &address)
     if(!model)
         return false;
 
-    // Fill in label from address book, if address has an associated label
+    // Fill in label from address book
     QString associatedLabel = model->getAddressTableModel()->labelForAddress(address);
-    if(!associatedLabel.isEmpty())
+    bool res = QString::compare(ui->addAsLabel->text(), associatedLabel, Qt::CaseSensitive) != 0;
```

Comment:
Why don't you pack the `QString::compare(ui->addAsLabel->text(), associatedLabel, Qt::CaseSensitive)` into the if:
`if (QString::compare(ui->addAsLabel->text(), associatedLabel, Qt::CaseSensitive))`

And why we need a `::compare` here? Wouldn't it be sufficient to just remove the `if(!associatedLabel.isEmpty())`. There is no drawback in overwriting the label string with the same string IMO.